python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -504,13 +504,22 @@ def queue_monitor_streaming(self) -> None:
     @structured_config_contributor
     def management_api_http(self) -> None:
         """management_api_http set based on management_eapi data-model."""
+        # Flag indicating if we are in ACT Digital Twin mode and if eAPI access in default VRF is enforced
+        act_backdoor_eapi_management: bool = (
+            self.shared_utils.digital_twin and self.inputs.digital_twin.environment == "act" and self.inputs.digital_twin.fabric.act_backdoor_management
+        )
```

Comment:
Also consider moving this to a property.
```suggestion
        # Flag indicating if we are in ACT Digital Twin mode and if eAPI access in default VRF is enforced
        act_preserve_management_access: bool = (
            self.shared_utils.digital_twin and self.inputs.digital_twin.environment == "act" and self.inputs.digital_twin.fabric.act_backdoor_management
        )
```

Reply:
Converted to cached_property